### PR TITLE
EUI-8679/8770: WorkbasketFiltersComponent and JudicialUser field fixes

### PR DESCRIPTION
**Pull request by Daniel-Lam**
### JIRA link (if applicable) ###
EUI-8769
EUI-8770

### Change description ###
EUI-8679: Remove JudicialUser FormControl values from the overall FormGroup value stored and passed as search criteria by the `WorkbasketFiltersComponent` when applying filters; EUI-8770: Ensure the `WorkbasketFiltersComponent` announces the selected jurisdiction and case type via the `JurisdictionService`; in the `WriteJudicialUserField` component, get the current jurisdiction and case type from the `JurisdictionService` if no case info is present.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by johnbenjamin-hmcts — APPROVED** [review]
LGTM

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/case-editor.module.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { NgModule } from '@angular/core';
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { RouterModule } from '@angular/router';
 import { RpxTranslationModule } from 'rpx-xui-translation';
+import { PaletteModule } from '../palette/palette.module';
```
Comment (connorpgpmcelroy) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/case-editor.module.ts:7]: Import ordering (why has this been moved)

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/create-case-filters/create-case-filters.component.ts [hunk 2]
```diff
@@ -8,6 +8,7 @@ import { DefinitionsService } from '../../services/definitions/definitions.servi
 import { OrderService } from '../../services/order/order.service';
 import { SessionStorageService } from '../../services/session/session-storage.service';
 import { CreateCaseFiltersSelection } from './create-case-filters-selection.model';
+import { JurisdictionService } from '../../services';
```
Comment (connorpgpmcelroy) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/create-case-filters/create-case-filters.component.ts:11]: Import ordering

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
2 minor changes requested

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/case-editor.module.ts (hunk 1 above)
Comment (Daniel-Lam) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/case-editor.module.ts:7]: Good spot, @connorpgpmcelroy - it came in from @andywilkinshmcts's work on EXUI-623 but I'll fix it

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Review by connorpgpmcelroy — DISMISSED** [review]
LGTM

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Pull request merged**